### DISCUSSION TITLE
multiple inverters

### DISCUSSION THREAD

**Discussion by JavacHolt** [slipx06/sunsynk-power-flow-card#391] [Q&A]
Hi
We have two systems, so two inverters, how do I go around adding both, or is that not yet currently possible?

**Comment by slipx06** [comment]
Hi. If they are two independent systems then you need a separate card for each. If they are setup as master/slave pair then you need to create template sensors as its not possible at the moment to display this on one card.

**Comment by Pho3niX90** [comment]
#477 #374 #90 

Just tagging the above issues, so that all interested could follow. 

I will have a look at adding arrays of sensors to either sum/avg depending on the type of data. Then there is no need to visualize extras. If visualization is needed, then the wide card would solve that, but for those that do not particularly care, this should solve it.
